templates/node.js-functionapp-on-linux.yml

```diff
@@ -0,0 +1,78 @@
+# Node.js Function App on Linux
+# Build and deploy a Node.js Function app on Linux.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+variables:
+
+  # Azure Resource Manager connection created during pipeline creation
+  azureSubscription: '{{ azureRmConnection.Name }}'
+
+  # Function app name
+  functionAppName: '{{ functionAppName }}'
+
+  # Agent VM image name
+  vmImageName: 'ubuntu-latest'
+
+stages:
+- stage: Build
+  displayName: Build stage
+  jobs:  
+  - job: Build
+    displayName: Build job
+    pool:
+      vmImage: $(vmImageName)
+      
+    steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '10.x'
+      displayName: 'Install Node.js'
+
+    - script: |
+        dotnet build --runtime ubuntu.16.04-x64 --output './bin/'

```

Comment:
Will it fail if there is not any .csproj or .sln file in the repo? Please check!

Reply:
The script logs an error, but the step doesn't fail and the pipeline proceeds. @azooinmyluggage to finalize - whether to keep this in the template or not ?

Reply:
@azooinmyluggage - do we need to use NodeTool@0 task here, just a simple install script one liner will do?

Reply:
@RoopeshNair just a best practice. Ensure there is a one node version in use unless changed deliberately.

Reply:
Proceeding with the check-in. Created an internal tracking item to verify whether we need it or not.